assets/src/components/routeLadder.tsx

```diff
@@ -213,6 +214,11 @@ const RouteLadder = ({
 
   const displayCrowding = someVehicleHasCrowding(vehiclesAndGhosts, route.id)
 
+  const screenSize = useScreenSize()
```

Comment:
can we do this within CSS instead?

Reply:
We could - I think that would make it a bit more difficult to test, but perhaps that's okay?

Reply:
Yeah I'm not really sure we'd be able to test it until we have CSS in Jest tests, but that's fine with me. We could assert a certain class is on it, I think we just need to change it to `d-none d-sm-block`, or something like that https://getbootstrap.com/docs/5.3/utilities/display/ (see the `Hidden only on xs` text)

Reply:
@firestack I updated to match ☝️, but I'm not sure that I love the lack of testability. I'll let it sit with me over the evening.

Reply:
@firestack I've let it sit with me, and while I don't _love_ relying on snapshot tests, they will solve this issue.

Ready for re-review!